internal/queries/volume.go

```diff
@@ -0,0 +1,94 @@
+package queries
+
+import (
+	"fmt"
+
+	"cloud.google.com/go/bigquery"
+	"google.golang.org/api/iterator"
+)
+
+// createVolumeTradeQuery returns a query that gets the the total volume to/from an asset, grouped by ledger.
+// The volume is calculated by looking at trades involving the assetswithin the provided ledger range.
+func createVolumeTradeQuery(asset Asset, volumeFrom bool, startLedger, endLedger string) string {
+	/*
```

Comment:
Little visually jarring to alternate between `//` and `/*...*/` for multiline comments. Small nit to change these in-function multiline comments to use `//`. Unless this separation is a Go practice I'm unaware of?